sql/create_train_val.go

```diff
@@ -64,27 +64,27 @@ func releaseTrainAndValDataset(ds *trainAndValDataset) {
 
 func createMaxcomputeDataset(db *DB, slct string, trainingUpperbound float32) (*trainAndValDataset, error) {
 	ds := namingTrainAndValDataset()
-	// create a table, then split it into 2 views
+	// create a table, then split it into train and val tables
 	stmt := fmt.Sprintf("CREATE TABLE %s LIFECYCLE %d AS SELECT *, RAND() AS %s FROM (%s) AS %s_ori", ds.table, temporaryTableLifecycle, randomColumn, slct, ds.table)
 	if _, e := db.Exec(stmt); e != nil {
 		log.Errorf("create temporary table failed, stmt:[%s], err:%v", stmt, e)
 		return nil, e
 	}
 	trainingCond := fmt.Sprintf("%s < %f", randomColumn, trainingUpperbound)
-	if e := createView(db, ds.table, ds.trainingView, trainingCond); e != nil {
+	if e := createMaxcomputeTable(ds.training, ds.table, db, trainingCond); e != nil {
 		return nil, e
 	}
 	validationCond := fmt.Sprintf("%s >= %f", randomColumn, trainingUpperbound)
-	if e := createView(db, ds.table, ds.validationView, validationCond); e != nil {
+	if e := createMaxcomputeTable(ds.validation, ds.table, db, validationCond); e != nil {
 		return nil, e
 	}
 	return ds, nil
 }
 
-func createView(db *DB, table, view, where string) error {
-	stmt := fmt.Sprintf("CREATE VIEW %s AS SELECT * FROM %s WHERE %s", view, table, where)
+func createMaxcomputeTable(target, origin string, db *DB, cond string) error {
```

Comment:
Is there a way to drop the temp table after the training is finished?

Reply:
PR [release resources](http://gitlab.alipay-inc.com/Arc/sqlflow/blob/develop/docker_alps/Dockerfile) will take over it, but I would like to use [LIFECYCLE](https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/training_and_validation.md#release-the-temporary-table) to handle it for Maxcompute at this time.